test-suite/shared/src/test/scala/org/scalajs/testsuite/javalib/lang/ClassTest.scala

```diff
@@ -55,11 +55,14 @@ class ClassTest {
     assertNotSame(classOf[java.lang.Double], java.lang.Double.TYPE)
   }
 
+  object TestObject
+
   @Test def getSimpleName(): Unit = {
     assertEquals("Integer", classOf[java.lang.Integer].getSimpleName())
     assertEquals("Class", classOf[java.lang.Class[_]].getSimpleName())
     assertEquals("Map", classOf[scala.collection.Map[_, _]].getSimpleName())
     assertEquals("InnerClass", classOf[ClassTestClass#InnerClass].getSimpleName())
+    assertEquals("TestObject$", TestObject.getClass.getSimpleName)
```

Comment:
The call to `getSimpleName` should have `()` . Could you amend the commit and force-push, otherwise it's good to go.

Reply:
Done